sprite.go

```diff
@@ -362,12 +416,36 @@ func (p *Sprite) OnCloned__1(onCloned func()) {
 	})
 }
 
+func (p *Sprite) fireTouchBegin(obj *Sprite) {
+	if p.hasOnTouchBegin {
+		p.doWhenTouchBegin(p, obj)
+	}
+}
+
 func (p *Sprite) fireTouched(obj *Sprite) {
 	if p.hasOnTouched {
 		p.doWhenTouched(p, obj)
 	}
 }
 
+func (p *Sprite) fireTouchEnd(obj *Sprite) {
+	if p.hasOnTouchEnd {
+		p.doWhenTouchEnd(p, obj)
+	}
+}
+
+func (p *Sprite) OnTouchBegin(onTouchBegin func(obj *Sprite)) {
```

Comment:
Since `begin` and `start` are nearly equivalent, I recommend using `start`, as it is already present in existing APIs like `OnStart`, `StartBackdrop`, and `aniConfig.OnStart`.

Reply:
OnTouchBegin will be replaced with OnTouchStart.